lib/ansible/modules/cloud/amazon/lambda_facts.py

```diff
@@ -155,7 +158,7 @@ def alias_details(client, module):
             if e.response['Error']['Code'] == 'ResourceNotFoundException':
                 lambda_facts.update(aliases=[])
             else:
-                module.fail_json(msg='Unable to get {0} aliases, error: {1}'.format(function_name, e))
+                module.fail_json_aws(e, msg="Trying to get aliases")
     else:
         module.fail_json(msg='Parameter function_name required for query=aliases.')
```

Comment:
Would it make sense to allow fail_json_aws the option of not having an exception, so this could also use fail_json_aws()? I like that fail_json_aws requires an exception because nagging people to include it isn't fun, but since fail_json() is still needed for non traceback errors I wonder if the mixture will be confusing/misused (and nagging to use the proper exception handling will still be necessary).